jclklib/proxy/connect.cpp

```diff
@@ -228,6 +228,12 @@ int Connect::connect()
     }
     m_sk.reset(sku);
 
+    while (system("ls /var/run/ptp4l*") != 0) {
```

Comment:
For good code practice, do init the pe.ptp4l_id  to zero. 
Btw, i didnt see the pe structure being initialized to init values , before being used. 
Maybe we can have the init somewhere before  Connect in proxy started / or before this is used : ptp4l_event_loop

Reply:
JClkLibCommon::ptp_event pe = { 0, {0, 0, 0, 0, 0, 0, 0, 0}, 0 , 0 };